DisguiseUnityRenderStream/Runtime/DisguisePresenter.cs

```diff
@@ -0,0 +1,233 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Disguise.RenderStream.Utils;
+using UnityEngine;
+using UnityEngine.Assertions;
+
+namespace Disguise.RenderStream
+{
+    /// <summary>
```

Comment:
Brainstorm: This is brilliant to understand when we have a problem I tried and love it!  However does that have any other usage than debug?  Will it be used in production?  If no then:
1. Should we give a name making it more obvious that this is for debugging?  UnityDebugPresenter? (Unity vs Disguise because the debug is displayed in the unity window on the rendering computer)
2. Do we want this debug to be present by default (should probably ask Kylo?)?  I'm asking because generally debugging in software are off by default.

Reply:
1. Good point. Any preference between `UnityDebugPresenter` and `UnityWindowPresenter`? I like "window" because it removes ambiguity around "local display" or "local screen".
2. The default texture selection is "None" so it's opt-in even when it's included in the schema. So the user wouldn't have to make a new build when they want to enable it.

Reply:
1. Personally I like to have Debug in there so it is obvious that it is for debugging. What about `UnityDebugWindowPresenter`?  The best of both world?
2. Is there potential draw back of selecting something?  I think it has a small performance impact?  What about user activate it, forget about it and then has worst performance than before and can't find why?